ci-operator/step-registry/aws/provision/cco-manual-users/static/aws-provision-cco-manual-users-static-commands.sh

```
@@ -57,24 +57,26 @@ function aws_create_user()
-function b64() { echo -n "${1}" | base64 ; }
-
 function create_secret_file_for_aws()
 {
     local ns=$1
     local name=$2
-    local b64_key_id=$3
-    local b64_key_sec=$4
+    local key_id=$3
+    local key_sec=$4
     local output_file=$5
     cat <<EOF >${output_file}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)